tests/integration/devfile/cmd_devfile_debug_test.go

```diff
@@ -31,142 +31,132 @@ var _ = Describe("odo devfile debug command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("odo debug on a nodejs:latest component", func() {
-		It("check that machine output debug information works", func() {
+	When("creating an application using devfile with debugrun", func() {
+		BeforeEach(func() {
 			helper.Cmd("odo", "create", "--project", commonVar.Project, componentName, "--context", commonVar.Context, "--devfile", helper.GetExamplePath("source", "devfiles", "nodejs", "devfile-with-debugrun.yaml")).ShouldPass()
 			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
 			helper.Cmd("odo", "push", "--debug", "--context", commonVar.Context).ShouldPass()
```

Comment:
Please remove this.

Reply:
@valaparthvi what do you want me to remove?

Reply:
`helper.Cmd("odo", "push", "--debug", "--context", commonVar.Context).ShouldPass()` This.